interface/src/app/sampleBin/addItemAnalyze.component.js

```diff
@@ -0,0 +1,43 @@
+angular.module('adage.sampleBin.addItemAnalyze', [
+  'adage.analyze.sampleBin',  // TODO fix after sampleBin is fully refactored
+  'adage.utils'
+])
+
+.component('addItemAnalyzeButton', {
+  templateUrl: 'sampleBin/addItemAnalyze.component.tpl.html',
+  bindings: {
+    item: '<'
+  },
+  controller: ['SampleBin', 'MlModelTracker', '$log', '$state',
+    function(SampleBin, MlModelTracker, $log, $state) {
+      var ctrl = this;
+      ctrl.sb = SampleBin;
+
+      ctrl.addItemAnalyze = function() {
+        SampleBin.addItem(ctrl.item);
+        $state.go('analysis-detail', {'mlmodel': MlModelTracker.id});
+      };
+
+      ctrl.itemTooltip = function() {
+        if (!ctrl.item) {
+          return 'ititializing...';
```

Comment:
Typo: `initializing` instead of `ititializing`

Reply:
LOL. Got it... thanks!